e2e-tests/playwright/e2e/plugins/application-listener.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { expect, test } from "@playwright/test";
+import { UIhelper } from "../../utils/ui-helper";
+import { Common } from "../../utils/common";
+
+test.describe("Test ApplicationListener", () => {
+  let uiHelper: UIhelper;
+
+  test.beforeEach(async ({ page }) => {
+    const common = new Common(page);
+    uiHelper = new UIhelper(page);
+    await common.loginAsGuest();
+  });
+
+  test("Verify that the LocationListener logs the current location", async ({
+    page,
+  }) => {
+    page.on("console", (msg) => {
+      if (msg.type() === "log") {
+        expect(msg.text()).toContain("pathname: /catalog");
+      }
+    });
```

Comment:
As discussed on slack. I see two risks here:

1. Can you verify that this fails when the test application listener isn't installed? I see a risk that it doesn't call `page.on` in that case.
2. And then I see a risk that some "random other code" calls console.log and let this test crash in the future.

Maybe you can collect the log messages instead and then ensure that it was at least once called with this log message?